vernac/g_vernac.mlg

```diff
@@ -115,10 +115,28 @@ GRAMMAR EXTEND Gram
     ]
   ;
   vernac_poly:
-    [ [ IDENT "Polymorphic"; v = vernac_aux ->
+    [ [ IDENT "Polymorphic"; v = vernac_cumulative ->
```

Comment:
Why do we have these separate grammar entries _poly / _cumulative / etc instead of handling each as a special case for quoted_attributes btw?

Reply:
I don't know but I don't mind this because with Jim's [reworked documentation of attributes](https://coq.gitlab.io/-/coq/-/jobs/446953137/artifacts/_install_ci/share/doc/coq/sphinx/html/language/gallina-specification-language.html#attributes)) this is pretty easy to follow. But yeah, maybe implementing as you said would make things more straightforward, so I can do this change if you wish.

Reply:
For instance that doc is incorrect as you can have multiple `legacy_attr` (eg Local Polymorphic) if you put them in the right order.
Putting them as special cases of quoted_attr would make the order not matter which is nicer.

Reply:
That doc shows precisely that you can write `Local Polymorphic` but not `Local Global` and not `Polymorphic Local`. But yes, I'll move everything to `quoted_attr`.

Reply:
Right I read it wrong

Reply:
@SkySkimmer So I tried, and all I managed to do was to broke proof mode parsing. I let you figure this one out if you want to pursue this idea: https://github.com/Zimmi48/coq/commit/6cd27fd95259536e0574a9a03a4b1b7f887f9ea4

Reply:
New version includes support for specifying legacy attributes in any order, but only after the quoted attributes. This did not break proof mode parsing.